File: src/tools/log.ts
import chalk from "chalk";

const log = console.log;

export function info(message: string) {
  log(chalk.blue(message));
}

export function success(message: string) {
  log(chalk.green(message));
}

export function warn(message: string) {
  log(chalk.hex("#FFA22E")(message));
}

export function error(message: string) {
  log(chalk.red(message));
}

File: prototypes/go-tasks-cli/log.go
package main

import (
	"fmt"

	"github.com/fatih/color"
)

func Log(message string) {
	fmt.Println(message)
}

func LogInfo(message string) {
	color.Blue("%s", message)
}

func LogHighlight(message string) {
	color.Magenta("%s", message)
}

func LogSuccess(message string) {
	color.Green("Success: %s", message)
}

func LogError(message string) {
	color.Red("Error: %s", message)
}

func LogWarn(message string) {
	color.Yellow("Warning: %s", message)
}
